src/fa/replayserver.py

```diff
@@ -74,7 +74,7 @@ def readDatas(self):
             #This prefix means "P"osting replay in the livereplay protocol of FA, this needs to be stripped from the local file            
             if data.startsWith("P/"):
                 rest = data.indexOf("\x00") + 1
-                self.__logger.info("Stripping prefix '" + str(data.left(rest)) + "' from replay.")
+                self.__logger.info("Stripping prefix '" + str(data.left(rest) - 1) + "' from replay.")
```

Comment:
Arguably it only happens once per replay, but that conversion still feels less tidy to me than just working with what QByteArray has.